app/gui/src/dashboard/utilities/write.ts

```diff
@@ -0,0 +1,14 @@
+/** @file Functions related to writing values to objects. */
+
+/**
+ * "Unsafe" because it bypasses React Compiler checks.
```

Comment:
I suppose it's a bug in react-compiler-eslint. Guess if we pass a component, that mutates global values into Compiler playground, it'll optimize that